packages/flutter/lib/src/rendering/semantics.dart

```diff
@@ -341,17 +341,26 @@ class SemanticsNode extends AbstractNode with DiagnosticableTreeMixin {
     _markDirty();
   }
 
-  bool get _inheritedMergeAllDescendantsIntoThisNode => _inheritedMergeAllDescendantsIntoThisNodeValue;
-  bool _inheritedMergeAllDescendantsIntoThisNodeValue = false;
-  set _inheritedMergeAllDescendantsIntoThisNode(bool value) {
+  /// Whether this node merges its semantic information into an ancestor node.
+  bool get isMergedIntoParent => _isMergedIntoParent;
+  bool _isMergedIntoParent = false;
+  set isMergedIntoParent(bool value) {
     assert(value != null);
-    if (_inheritedMergeAllDescendantsIntoThisNodeValue == value)
+    if (_isMergedIntoParent == value)
       return;
-    _inheritedMergeAllDescendantsIntoThisNodeValue = value;
+    _isMergedIntoParent = value;
     _markDirty();
   }
```

Comment:
should these bools go in the flags?

Reply:
They are currently not transfered to the engine side and only processed on the framework side. That's why I've kept them separate from the flags for now.